apps/condo/domains/common/components/Table/Filters.tsx

```diff
@@ -191,17 +186,6 @@ export const getOptionFilterDropdown: GetOptionFilterDropdownType = ({ container
                     .map((option) => get(option, 'label'))
                     .filter(Boolean)
             }
-            if (!isEmpty(selectedValue)) {
-                logEvent({
-                    eventName,
-                    eventProperties: {
-                        component: {
-                            id: checkboxGroupProps.id,
-                            value: selectedValue,
-                        },
-                    },
-                })
-            }
```

Comment:
Is something planned instead?

Reply:
For this specific filter - I don't think so... 

I've removed some old logs because I my head they was not used / valuable enough to build conversion / other product dashboards

Reply:
And "events just for events" for me is a bit useless. 

I would prefer combination of "basic events" (UI-kit) + "business valuable events" (on which you can compute DAU or something like that)